jetty/src/test/scala/org/http4s/server/jetty/NotFoundHttpAppSpec.scala

```diff
@@ -0,0 +1,75 @@
+package org.http4s
+package server
+package jetty
+
+import java.net.{HttpURLConnection, URL}
+
+import cats.data.Kleisli
+import cats.effect.IO
+import org.http4s.dsl.io._
+
+import scala.io.Source
+import scala.util.Try
+
+class NotFoundHttpAppSpec extends Http4sSpec {
```

Comment:
We no longer need this spec - it doesn't test anything that isn't covered elsewhere.

Reply:
Ok, I removed it